transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/FileTransformations.scala

```diff
@@ -31,16 +31,18 @@ object FileTransformations {
   private val logger = LoggerFactory.getLogger(getClass)
 
   /** Divide a stream of raw files into branches, with one branch per file category. */
-  def partitionRawFiles(rawStream: SCollection[Msg]): FileBranches[SCollection, Msg] = {
+  def partitionRawFiles(
+    rawStream: SCollection[(Msg, Option[Msg])]
```

Comment:
It feels a little weird to be passing the experiments into this method where they're not being used, but I see why you are doing it since joining the experiments later on would be messy.

Reply:
If we add any more JOINs (i.e. for pipelines), I think the move will be to declare a new `case class` that wraps up the raw file and all of its linked metadata, and pass those around through these methods. For now this is OK.